app/views/candidate_interface/guidance/index.html.erb

```diff
@@ -32,7 +32,7 @@
           <div class="app-timeline__log-card">
             <div class="app-timeline__log-card__note">
               <p class="app-timeline__log-card__note--title">
-                <%= t('.find_courses_html', link: govuk_link_to('finding postgraduate teacher training courses', find_url)) %>
+                <%= t('.find_courses_html', link: govuk_link_to('finding teacher training courses', find_url)) %>
```

Comment:
Unrelated change, I got tagged in a slack conversation to ask if I could fix it. Cannot be bothered to do a separate PR.